### PR TITLE
adds UST indicator

### PR DESCRIPTION
**Pull request by emma-nechamkin**

### CODE REVIEW

**Comment by emma-nechamkin** [comment]
I am going to have to update the fixtures i believe

File: data/data-pipeline/data_pipeline/score/field_names.py [hunk 1]
```diff
@@ -170,7 +170,7 @@
 NPL_FIELD = "Proximity to NPL sites"
 AIR_TOXICS_CANCER_RISK_FIELD = "Air toxics cancer risk"
 RESPIRATORY_HAZARD_FIELD = "Respiratory hazard index"
-UST_FIELD = "Underground storage tanks"
+UST_FIELD = "Leaky underground storage tanks"
```
Comment (vim-usds) [inline data/data-pipeline/data_pipeline/score/field_names.py:173]: @emma-nechamkin and @KameronKerger - can these also be used for the sidepanel copy?

**Review by vim-usds — COMMENTED** [review]
LGTM!

**Comment by KameronKerger** [comment]
working on the side panel today with @emma-nechamkin - will post when done.

**Comment by vim-usds** [comment]
> working on the side panel today with @emma-nechamkin - will post when done.

Awesome! Looking forward to it when the time is right!

**Review by esfoobar-usds — APPROVED** [review]
🍏

**Pull request merged**